for_other_cantons.py

```diff
@@ -154,9 +152,10 @@ def plot_predictions_canton(table_name, curve, canton, full_name_canton, title =
     target_curve_name = curve
     
     
-    df_val = pd.read_sql_table(table_name, engine, schema = 'switzerland', index_col = 'date')
+    df_val = pd.read_sql(f"select * from switzerland.{table_name} where canton='{canton}';", engine)
```

Comment:
Don't use `select *` here, because it will pull the entire table. select only the columns we are going to use, for example: `select datum, entries, geoRegion from...`